pom.xml

```diff
@@ -503,6 +503,7 @@
                 <module>http/graphql-telemetry</module>
                 <module>http/vertx</module>
                 <module>http/management</module>
+                <module>http/grpc</module>
```

Comment:
Do you fancy to update project README with new module?

Reply:
We have a separate readme for this module. Is it not enough?

Reply:
Here https://github.com/quarkus-qe/quarkus-test-suite/blob/main/README.md#httphttp-advanced is documented gRPC converage that transitively applies on `http-advanced-reactive` module. Personally I don't need any README, but as one is already there, I'd like to keep it up to date or drop it completely.

Reply:
@michalvavrik
The thing is, some GRPC coverage (fairly basic) remains in http-advanced(x2), since 
1. new gRPC is only concerned  with reactive
2. integration between many http modules probably deserves a separate coverage anyway

I only moved tests, implemented by Georgii since they were reactive-only.
And by the way, @gtroitsk , I still wait for confirmation, that you are fine with it.